pipelines/deploy-fbc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1
```

Comment:
do we need to version?

Reply:
I removed it, thanks! I temporarily added this Pipeline for testing purposes. It will not be a part of this PR after testing is done.

Reply:
actually, it looks like it's needed. I got this error when I removed it:

> resolver failed to get Pipeline : invalid runtime object: Object 'apiVersion' is missing in 'kind: Pipeline

Reply:
no, sorry, my bad. Do you need to version the pipeline?

Reply:
> Do you need to version the pipeline?
Could you clarify what you mean by that? I just followed these examples when creating `deploy-fbc` pipeline: https://github.com/konflux-ci/pipeline-samples/tree/main/pipelines/integration

Reply:
miss this...

pipelines/deploy-fbc/0.1/deploy-fbc.yaml?

Reply:
Ah, got it! I'll add this when I create a PR with the full pipeline. Thank you!